### PR TITLE
build for v0.12.0 release

### PR DESCRIPTION
**Pull request by grlee77**

### CODE REVIEW

**Comment by grlee77** [comment]
@hgomersall, please approve when you get a chance
Hopefully once we merge the wheels for Python 3.5-3.8 will show up at wheels.scipy.org

**Comment by grlee77** [comment]
Okay, wheels for 2.7 are starting to show up on wheels.scipy.org.  Please also approve/merge this branch and then I can transfer them to PyPI, create a release on GitHub and update the conda-forge recipe.

**Review by hgomersall — APPROVED** [review]

**Comment by hgomersall** [comment]
Marvellous. Well done and thanks!

**Pull request merged**
